design-principles.md

```diff
@@ -61,12 +61,15 @@ comments on other people's PRs. The more reviews the better.
 Reviewers should carefully read the code and make sure they understand
 it. A reviewer should check the code for both high level concerns,
 such as whether the code's structure makes sense, as well as
-readability and adherence to the style guide. Reviewers should not
-give an LGTM unless the patch has tests that verify all the affected
-code, or unless a test would make no sense. If you review a patch, you
-are sharing the responsibility for the patch with its author. You
-should only give an LGTM if you would feel confident answering
-questions about the code.
+readability and adherence to the [relevant style guide](https://google.github.io/styleguide/).
```

Comment:
style guide should line to https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo